.vscode/extensions.json

```diff
@@ -5,6 +5,7 @@
 	"styled-components.vscode-styled-components",
```

Comment:
We should probably add `.vscode/` to `.gitignore`

Reply:
Good call, added.